src/controllers/controller.polarArea.js

```diff
@@ -23,6 +22,23 @@ export default class PolarAreaController extends DatasetController {
     };
   }
 
+  parseObjectData(meta, data, start, count) {
+    const {iScale} = meta;
+    const {key = 'key'} = this._parsing;
+    const parsed = new Array(count);
+    let i, ilen, index, item;
+
+    for (i = 0, ilen = count; i < ilen; ++i) {
+      index = i + start;
+      item = data[index];
+      parsed[i] = {
+        r: iScale.parse(resolveObjectKey(item, key), index)
+      };
```

Comment:
```suggestion
      parsed[i] = iScale.parse(resolveObjectKey(item, key), index);
```

Reply:
Not sure if that is correct, when I logged the normal implementation (`parsePrimitiveValue`) it also putted it in an array of objects where the key is `r`. 

And in the `getLabelAndValue` method it also tries to read the `r` value in the object for its value: https://github.com/chartjs/Chart.js/blob/ec67123df890a8eaa18c5b60f980a94644988536/src/controllers/controller.polarArea.js#L18

Reply:
ok, my suggestion is wrong